src/Abc.Zebus/Directory/PeerDirectoryClient.cs

```diff
@@ -368,6 +434,8 @@ public void Handle(PeerSubscriptionsForTypesUpdated message)
             peer.Value.SetSubscriptionsForType(message.SubscriptionsForType ?? Enumerable.Empty<SubscriptionsForType>(), message.TimestampUtc);
 
             PeerUpdated?.Invoke(message.PeerId, PeerUpdateAction.Updated);
+
+            DispatchSubscriptionUpdatedMessages(message.PeerId, message.SubscriptionsForType); // TODO
```

Comment:
What's this TODO?

Reply:
It's recursive - one must delete the TODO